File: Easy/2026-03-05-1758-Minimum-Changes-To-Make-Alternating-Binary-String/solution.py
class Solution:
    def minOperations(self, s: str) -> int:
        count1 = count2 = 0
        for i in range(len(s)):
            expected_char1 = '0' if i % 2 == 0 else '1'
            expected_char2 = '1' if i % 2 == 0 else '0'
            if s[i] != expected_char1:
                count1 += 1
            if s[i] != expected_char2:
                count2 += 1
        return min(count1, count2)

File: Easy/2026-03-05-1758-Minimum-Changes-To-Make-Alternating-Binary-String/solution.js
var minOperations = function(s) {
    let count1 = 0, count2 = 0;
    
    for (let i = 0; i < s.length; i++) {
        const expectedChar1 = i % 2 === 0 ? '0' : '1';
        const expectedChar2 = i % 2 === 0 ? '1' : '0';
        
        if (s[i] !== expectedChar1) count1++;
        if (s[i] !== expectedChar2) count2++;
    }
    
    return Math.min(count1, count2);
};